src/main/java/de/adesso/service/ConfigService.java

```diff
@@ -43,43 +42,44 @@
     private String FIRSTSPIRIT_XML_PATH;
 
     public void checkConfiguration() {
-        /*
-            REPO URL
-         */
+        // Repo URL
         if (REPOSITORY_REMOTE_URL == null) {
-            LOGGER.error("ERROR: Environment variable not provided: REPOSITORY_REMOTE_URL \n" +
-                    "Please provide something in the format: https://github.com/myAccount/devblog");
-            System.exit(1);
+            LOGGER.error("Environment variable not provided: REPOSITORY_REMOTE_URL. " +
+                    "Please provide something in the format: https://github.com/myAccount/devblog.");
+            LOGGER.error("Exiting jekyll2cms.");
+            System.exit(10);
         } else {
-            LOGGER.info("SUCCESS: Environemnt variable provided: REPOSITORY_REMOTE_URL");
+            LOGGER.info("Environment variable provided: REPOSITORY_REMOTE_URL");
         }
 
-        /*
-            Github-Names
-         */
+        // GitHub name
         if (GIT_AUTHOR_NAME == null) {
-            LOGGER.error("ERROR: Environment variable not provided: GIT_AUTHOR_NAME \n" +
-                    "Please provide some Github-Username");
-
-            System.exit(1);
+            LOGGER.error("Environment variable not provided: GIT_AUTHOR_NAME. " +
+                    "Please provide some Github-Username.");
+            LOGGER.error("Exiting jekyll2cms.");
+            System.exit(11);
         } else {
-            LOGGER.info("SUCCESS: Environemnt variable provided: GIT_AUTHOR_NAME");
+            LOGGER.info("Environment variable provided: GIT_AUTHOR_NAME");
         }
 
+        // GitHub author mail
         if (GIT_AUTHOR_MAIL == null) {
-            LOGGER.error("ERROR: Environment variable not provided: GIT_AUTHOR_MAIL \n" +
-                    "Please provide some correspronding Email to the Github Username");
-            System.exit(1);
+            LOGGER.error("Environment variable not provided: GIT_AUTHOR_MAIL. " +
+                    "Please provide the provided GitHub user's Email.");
```

Comment:
```suggestion
                    "Please provide the email of the specified GitHub user.");
```

An der Stelle: Warum brauchen wir hier die GitHub assoziierte email?